src/pages/accounts/AccountDetails.tsx

```diff
@@ -1,451 +1,279 @@
-                        Failed to load!
-                    </Alert>
-                </Snackbar >
-            })
-    }
-    const accountCountry = (country: string) => {
-        let countryName: string[] | undefined;
-        for (countryName of countries) {
-            if (Array.isArray(countryName) && countryName.includes(country)) {
-                const ele = countryName;
-                break;
-            }
-        }
-        return countryName?.[1]
-    }
-    const editHandle = () => {
-        // navigate('/contacts/edit-contacts', { state: { value: contactDetails, address: newAddress } })
-        let country: string[] | undefined;
-        for (country of countries) {
-            if (Array.isArray(country) && country.includes(accountDetails?.country || '')) {
-                const firstElement = country[0];
-                break;
-            }
-        }
-        navigate('/app/accounts/edit-account', {
-            state: {
-                value: {
-                    ...accountDetails,
-                    name: accountDetails?.name,
-                    phone: accountDetails?.phone,
-                    email: accountDetails?.email,
-                    billing_address_line: accountDetails?.billing_address_line,
-                    billing_street: accountDetails?.billing_street,
-                    billing_city: accountDetails?.billing_city,
-                    billing_state: accountDetails?.billing_state,
-                    billing_postcode: accountDetails?.billing_postcode,
-                    billing_country: accountDetails?.billing_country,
-                    contact_name: accountDetails?.contact_name,
-                    teams: accountDetails?.teams || [],
-                    assigned_to: accountDetails?.assigned_to || [],
-                    tags: accountDetails?.tags || [],
-                    account_attachment: accountDetails?.account_attachment || null,
-                    website: accountDetails?.website,
-                    status: accountDetails?.status,
-                    lead: accountDetails?.lead?.title || '',
-                    contacts: accountDetails?.contacts || [],
-        
-                }, id: state?.accountId,
-                contacts: state?.contacts || [], status: state?.status || [], tags: state?.tags || [], users: state?.users || [], countries: state?.countries || [], teams: state?.teams || [], leads: state?.leads || []
-            }
-        }
-        )
-    }
+      const Header = {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+        Authorization: localStorage.getItem('Token') ?? '',
+        org: localStorage.getItem('org') ?? '',
+      };
 
-    const backbtnHandle = () => {
-        navigate('/app/accounts')
-    }
+      try {
+        const res = await fetchData(
+          `${AccountsUrl}/${accountId}/`,
+          'PATCH',
```

Comment:
In order for this request to work, you need to add a PATCH endpoint to the views.py in accounts, under `AccountDetailsView`
Something like this which I added before to Deals:
```python
    @extend_schema(
        tags=["Deals"],
        parameters=swagger_params1.organization_params, request=DealCreateSwaggerSerializer
    )
    def patch(self, request, pk, format=None):
        deal = self.get_object(pk)
        serializer = DealSerializer(deal, data=request.data, partial=True)  # Allow partial updates
        if serializer.is_valid():
            serializer.save()
            return Response(serializer.data)
        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
``` 

Patch allows partial changes, like here we want to only change the description and it's not a sync change with the other parts 🙂
You can check views.py under deals to see the structure.